language/move-vm/runtime/src/data_cache.rs

```diff
@@ -197,7 +199,20 @@ impl<'r, 'l, C: RemoteCache> DataStore for TransactionDataCache<'r, 'l, C> {
 
                     GlobalValue::cached(val)?
                 }
-                None => GlobalValue::none(),
+                Ok(None) => GlobalValue::none(),
+                Err(err) => {
+                    println!("HHIIIIIT");
+                    let msg = format!("Unexpected storage error: {:?}", err);
+                    // REVIEW: better way to get info out of a PartialVMError?
```

Comment:
I think it might be fair to add an all_data equivalent to `PartialVMError`. I never needed it so I never added it